include_core/omrport.h

```diff
@@ -972,7 +972,11 @@ typedef struct J9PortVmemIdentifier {
 	uintptr_t pageFlags;
 	uintptr_t mode;
 	uintptr_t allocator;
+#if defined(LINUX)
 	int fd;
+#elif (defined(WINDOWS) || defined(WIN32))
+	void *fd;
```

Comment:
Shouldn't this be HANDLE and not void on Windows?

Reply:
You're correct but to do that we need to import windows.h. If we use HANDLE I get the error `syntax error : identifier 'HANDLE'`. Then When I include `windows.h` I get more errors than I can print here, one of them is `error C2371: 'UDATA' : redefinition; different basic types`. But since HANDLE is defined as `typedef PVOID HANDLE` and PVOID is `typedef void *PVOID` I used `void *`